apps/meeting/scripts/setup_chime_react_submodule.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
```

Comment:
In JS SDK [demo/scripts](https://github.com/aws/amazon-chime-sdk-js/tree/main/demos/browser/script) we follow a dashed naming convention, can we follow the same here: `setup-chime-react-submodule.js`?

Reply:
Fixed.